src/steamship/agents/service/agent_service.py

```diff
@@ -87,9 +87,70 @@ def run_agent(self, agent: Agent, context: AgentContext):
             logging.info(f"Emitting via function: {func.__name__}")
             func(action.output, context.metadata)
 
+    def _prompt(self, prompt: str, **kwargs) -> List[Block]:
```

Comment:
what does this do that the other prompt doesn't?

Reply:
REmoved.